cfgov/cfgov/settings/base.py

```diff
@@ -360,6 +360,7 @@
                 os.getenv("ES_USER", "admin"),
                 os.getenv("ES_PASS", "admin"),
             ),
+            "use_ssl": True, 
```

Comment:
It looks like this setting is causing a problem running the unit tests and functional tests in GHA. 

I think we'll need to add a `OPENSEARCH_DSL["default"]["use_ssl"]: False` override to `cfgov/settings/test.py` for  unit tests to work.

For the functional tests... [the GitHub Action just calls `manage.py runserver`](https://github.com/cfpb/consumerfinance.gov/blob/main/.github/workflows/functional-tests.yml#L76), which [defaults to `cfgov/settings/local.py`](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/manage.py#L7), which... I don't think it is right. But changing that is out of scope here, so we'd need to add the same override to that file as well.

Reply:
I disabled `use_ssl` in both `local.py` and and `test.py`.  Appears that both tests are passing now.